text/0000-fabric-private-chaincode-lite.md

```diff
@@ -41,17 +42,37 @@ Such data includes secret cryptographic keys, which the chaincode uses to secure
     and also still mention all possible use-cases. 
     The distinction is done later in the architecture/design section
 -->
+
+***TODO*** document is missing a bit in the intro section about:
+- clear statement what is the purpose of the document
+- link to full FPC
```

Comment:
link to _RFC_ for full FPC? I.e., more in conjunction with below? (E..g, add a section at end of paragraph that this RFC superceed (for now) the old FPC and already introduces FPC lite?